test/src/test/kotlin/graphql/nadel/tests/next/fixtures/hydration/CastHydrationLongInputToStringForMatchingTestSnapshot.kt

```diff
@@ -0,0 +1,132 @@
+            ExpectedServiceCall(
+                service = "myService",
+                query = """
+                | {
+                |   someData {
+                |     __typename__batch_hydration__spaces: __typename
+                |     batch_hydration__spaces__spaceIds: spaceIds
+                |   }
+                | }
+                """.trimMargin(),
+                variables = "{}",
+                result = """
+                | {
+                |   "data": {
+                |     "someData": {
+                |       "batch_hydration__spaces__spaceIds": [
+                |         1,
+                |         2,
+                |         10
+                |       ],
+                |       "__typename__batch_hydration__spaces": "SomeData"
+                |     }
+                |   }
+                | }
+                """.trimMargin(),
+                delayedResults = listOfJsonStrings(
+                ),
+            ),
+            ExpectedServiceCall(
+                service = "myService",
+                query = """
+                | {
+                |   spaces(ids: [1, 2, 10]) {
+                |     id
+                |     batch_hydration__spaces__id: id
+                |   }
+                | }
+                """.trimMargin(),
+                variables = "{}",
+                result = """
+                | {
+                |   "data": {
+                |     "spaces": [
+                |       {
+                |         "id": "1",
+                |         "batch_hydration__spaces__id": "1"
+                |       },
+                |       {
+                |         "id": "2",
+                |         "batch_hydration__spaces__id": "2"
+                |       },
+                |       {
+                |         "id": "10",
+                |         "batch_hydration__spaces__id": "10"
+                |       }
+                |     ]
+                |   }
+                | }
+                """.trimMargin(),
+                delayedResults = listOfJsonStrings(
+                ),
+            ),
+        )
+
+    /**
+     * ```json
+     * {
+     *   "data": {
+     *     "someData": {
+     *       "spaces": [
+     *         {
+     *           "id": "1"
+     *         },
+     *         {
+     *           "id": "2"
+     *         },
+     *         {
+     *           "id": "10"
+     *         }
+     *       ]
+     *     }
+     *   }
+     * }
+     * ```
+     */
+    override val result: ExpectedNadelResult = ExpectedNadelResult(
+            result = """
+            | {
+            |   "data": {
+            |     "someData": {
+            |       "spaces": [
+            |         {
+            |           "id": "1"
+            |         },
+            |         {
+            |           "id": "2"
+            |         },
+            |         {
+            |           "id": "10"
+            |         }
```

Comment:
Before this would've been

```
"spaces": [null, null  nul]
```

The fix means the integer input now matches the result ID (String).